repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/pooling/ResourceContext.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.openrepose.commons.utils.pooling;
 
+@FunctionalInterface
 public interface ResourceContext<R, T> {
 
     T perform(R resource) throws ResourceContextException;
```

Comment:
Yeah, I'm not clear on what it does either.  I'm going to remove the annotation and suppress the warning.